### ISSUE TITLE
[Bug] Using Anti packet kick will kick you off the server

### ISSUE THREAD

**Issue by alfa333333** [MeteorDevelopment/meteor-client#4122]
### Describe the bug

Using Anti Packet Kick will kick you off the server
Internal Exception: io.netty.handler.codec.DecoderException: net.minecraft.class_8801: Tried to read NBT tag with too high complexity, depth > 0

### Steps to reproduce

```bash
When you turn it on it will kick you and not let you back on the server until you turn it off
```

### Meteor Version

meteor client v0.5.5 build #1944

### Minecraft Version

1.20.2

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by ProJakob** [comment]
Can confirm, same just happened to me

**Comment by Shiroiame-Kusu** [comment]
I also met this problem yesterday on my server.
didnt solve until I saw this issue.

**Comment by Wide-Cat** [comment]
should be fixed by 3a426d57c52c09ebc6f365905e2f6d2830febf47
